parser_library/src/processing/processing_manager.cpp

```diff
@@ -249,7 +250,27 @@ void processing_manager::finish_copy_member(copy_processing_result result)
     lsp_analyzer_.copydef_finished(member, std::move(result));
 }
 
-void processing_manager::finish_opencode() { lsp_analyzer_.opencode_finished(); }
+void processing_manager::finish_opencode()
+{
+    if (auto statements = opencode_prov_.get_preprocessor_statements())
+    {
+        for (const auto& stmt : *statements)
+        {
+            if (!stmt)
+                continue;
+
+            lsp_analyzer_.analyze(*stmt);
+
+            if (stmt->m_resemblence == context::id_storage::well_known::COPY)
+                asm_processor::process_copy(*stmt, ctx_, lib_provider_, nullptr, false);
+
+            // diagnosable_impl::add_diagnostic(diagnostic_s::fade(file_loc_, stmt->stmt_range_ref())); // todo create
```

Comment:
?

Reply:
This is going to be changed in one of the next PRs. For now, it's just a placeholder for me to not forget to route the "faded" statements via some other route than through regular diagnostics mechanism